enterprise/coderd/coderd.go

```diff
@@ -365,14 +366,22 @@ func (api *API) updateEntitlements(ctx context.Context) error {
 	api.entitlementsUpdateMu.Lock()
 	defer api.entitlementsUpdateMu.Unlock()
 
+	gitAuthConfigs := 0
```

Comment:
This logic is worthy of a comment, e.g.:

```go
// The built-in auth isn't licensed so we don't count it.
```

This is an easy rule to miss while refactoring.